docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::note
+The documentation is written with [docusaurus](https://docusaurus.io/). You don't need install anything specific for docusaurus but it can be helpful to reference their documentation if needed.
+:::
+
+:::note
+If you are working with the documentation from within VSCode+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll n eed to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+In order to view and verify documentation updates locally you'll need to install `node.js` and `npm`. Please see [nodejs.org](https://nodejs.org/) for details on how to install these tools.
```

Comment:
I typically see "Node.js" with a capital N (several examples in https://en.wikipedia.org/wiki/Node.js)